### ISSUE TITLE
Game stuck while taking off into orbit

### ISSUE THREAD

**Issue by MaixGit** [tinyhoot/ShipLobby#1]
I haven't tested it with multiple people yet but in a lobby by myself I can land on a moon but when I take off it gets stuck at "Waiting for ship to land", and never actually progresses from that.

These are the other mods I'm using which work perfectly fine and are up to date.
![image](https://github.com/tinyhoot/ShipLobby/assets/43832909/e3861d7d-f9bc-4963-878b-e35c04193eb4)

**Comment by MaixGit** [comment]
I've also tried with only ShipLobby installed and it's the same result, so it doesn't seem like it's mods that are conflicting

**Comment by DavryH** [comment]
Started having the same issue after installing the mod.

**Comment by dukedotdev** [comment]
Also having this issue, console outputs the below when attempting to go into orbit. In case it's relevant, this log was from a private lobby.
```
[Error  : Unity Log] NullReferenceException
Stack trace:
UnityEngine.MonoBehaviour.StartCoroutine (System.Collections.IEnumerator routine) (at <e27997765c1848b09d8073e5d642717a>:0)
ShipLobby.Patches.GameNetworkManagerPatcher.ReopenSteamLobby (StartOfRound __instance) (at <3d35ac6ab787486eb06bc3e0b2b78aa1>:0)
(wrapper dynamic-method) StartOfRound.DMD<StartOfRound::EndOfGame>(StartOfRound,int,int)
StartOfRound.EndOfGameClientRpc (System.Int32 bodiesInsured, System.Int32 daysPlayersSurvived, System.Int32 connectedPlayersOnServer) (at <39ddadb2aa5843e586a6c642ee60b0b5>:0)
StartOfRound+<unloadSceneForAllPlayers>d__239.MoveNext () (at <39ddadb2aa5843e586a6c642ee60b0b5>:0)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <e27997765c1848b09d8073e5d642717a>:0)
```

**Comment by harrisonhoward** [comment]
+1 - Same error message too.

**Comment by KlowNJ420** [comment]
+1 happens over here too

**Comment by BodenM** [comment]
+1 happening for me as well

**Comment by tinyhoot** [comment]
The cause was BepInEx's manager object getting deleted super early by the game's scene changes. Fixed in 8918dbc
